zebra-consensus/Cargo.toml

```diff
@@ -59,8 +59,9 @@ tracing = "0.1.39"
 tracing-futures = "0.2.5"
 
 orchard = "0.6.0"
+sapling = { package = "sapling-crypto", git = "https://github.com/oxarbitrage/sapling-crypto", branch = "add-verifying-key-methods" }
 
-zcash_proofs = { version = "0.13.0-rc.1", features = ["multicore" ] }
+zcash_proofs = { package = "zcash_proofs", git = "https://github.com/oxarbitrage/librustzcash", branch = "use-custom-sapling" }
```

Comment:
This just points to a custom version of zcash_proofs 0.14.0 that use the sapling version we modified above, Zebra has troubles if there are 2 versions of sapling running.